src/utils/selectors.ts

```diff
@@ -1,6 +1,12 @@
+import { AppState } from '../reducers/rootReducer'
 import { createSelector } from 'reselect'
 
 export const selectToken = createSelector(
-    (state: any) => state.token,
+    (state: AppState) => state.token,
     (token) => token
 )
+
+export const selectIsPendingToken = createSelector(
```

Comment:
do we really need to use reselect for that simple getters?

Reply:
you're right. It's overkill just for simple getter

Reply:
on my side I did since `selectToken` is reused, even if the selector is a simple one. I guess that's also a point to consider

Reply:
well, the goal of reselect if to optimize for big selectors with map/filters and aggregation. 
Here, in term of performance, I bet destructuring is more impactful than properties access on object :D

Reply:
I used that for [memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors)